python/ray/ml/checkpoint.py

```diff
@@ -1,117 +1,224 @@
-import abc
-from typing import Dict, Optional
-
-import ray
-
-
-class Checkpoint(abc.ABC):
-    """Checkpoint interface"""
-
-    def __init__(self, metadata: Optional[Dict] = None):
-        self.metadata = metadata or {}
+import shutil
```

Comment:
Hm assuming this checkpoint thing will go away?

Reply:
Yep, this is just a subset copied from @krfricke's PR.